src/relay/frontend/tvm_build_module.h

```diff
@@ -0,0 +1,108 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file relay/backend/compile_engine.h
+ * \brief Internal compilation engine handle function cache.
+ *  and interface to low level code generation.
+ */
+#ifndef TVM_RELAY_FRONTEND_TVM_BUILD_MODULE_H_
+#define TVM_RELAY_FRONTEND_TVM_BUILD_MODULE_H_
+
+#include <tvm/build_module.h>
+#include <tvm/codegen.h>
+#include <tvm/ir_pass.h>
+#include <tvm/operation.h>
+
+#include <string>
+
+#include "graph_runtime_codegen.h"
+#include "utils.h"
+
+namespace tvm {
+/*!
+ * \brief Build TVM Runtime
+ *
+ * \param funcs Lowered functions
+ * \param target Target device
+ * \param target_host Target host device
+ * \param config TVM Build config
+ * \return runtime::Module
+ */
+runtime::Module tvm_build(const Array<LoweredFunc>& funcs, const Target& target,
```

Comment:
Can't this just be an invocation of https://github.com/dmlc/tvm/blob/c53dd1020e24605c5cb6b38d73f7638386bb94fd/src/codegen/build_module.cc#L406-L485?

```
runtime::Module build(const Array<LoweredFunc>& funcs,
                      const Target& target,
                      const Target& target_host,
                      const BuildConfig& config)
```

?

Reply:
Good catch. We need to register this function for global access and delete this file.